@types/modules/pm/prize.d.ts

```diff
@@ -0,0 +1,26 @@
+declare interface PMPrizeUpdateRequest {
+    prizeDisplayName?: string;
+    prizePrice?: string;
+    prizeListingName?: string;
+    prizeUrl?: string;
+    prizeasin?: string;
```

Comment:
Can we do ASIN as all caps?
```suggestion
    prizeASIN?: string;
```